### DISCUSSION TITLE
How do I implement the Fixed-Header?

### DISCUSSION THREAD

**Discussion by GregorGander** [Power-Components/livewire-powergrid#970] [Themes]
What do I have to do, to implement the Fixed-Header functionality?

**Comment by aaronjonesxyz** [comment]
I'd also love to know, the docs are quite sparse and it would be great if the live demo included code snippets.

**Reply by luanfreitasdev** [reply]
https://demo.livewire-powergrid.com/header-fixed

**Reply by aaronjonesxyz** [reply]
There's no code on the example page, but I worked out how to do it by looking at the CSS and making a custom theme.

**Reply by luanfreitasdev** [reply]
https://github.com/Power-Components/powergrid-demo-misc/blob/main/app/Http/Livewire/HeaderFixedTable.php
